### ISSUE TITLE
Daytime color temperature

### ISSUE THREAD

**Issue by GnaXi** [xflux-gui/xflux-gui#14]
In the Windows version it is possible to have a daytime color temperature set as well.
It would be nice to see this introduced on Linux! :)

**Comment by skelic** [comment]
Good idea! On linux, whenever I set the color mode to be more thicker, it will slowly turn back to original brightness.

**Comment by ntc2** [comment]
Sounds good to me. Someone want to implement this and submit a pull request?

**Comment by skelic2** [comment]
Hi, anyway where can you find the xflux-gui Windows version source/branch code?
Someone could just integrate the specific code taken from Windows source code version into this branch, and modify it for gtk platform.

**Comment by GnaXi** [comment]
I do not believe that the original f.lux for Windows is open source.

**Comment by skelic2** [comment]
Oh, I see. I thought the Windows version is also a part of open source project.

**Comment by itsabear** [comment]
I want this feature so bad. Using redshift will always return `method rand` at `STDOUT`. Don't know how to make redshift working on Ubuntu.

**Comment by bknie1** [comment]
Here to show support for this request! I love working at night but I also liked having a daytime tint. Linux has been harder on my eyes.

**Comment by ntc2** [comment]
[Redshift](http://jonls.dk/redshift/) supports daytime temps. E.g. you can run the command line version with

```shell
redshift -v -t 4200:2500
```

to use color temp 4200 in the day time and 2500 at night, with automatic timezone-based transitions. It would be great to have this daytime temp feature in flux GUI, but in the mean time ...

**Comment by bknie1** [comment]
> Redshift supports daytime temps.

Redshift starts, rolls through each specific setting, and then returns to full brightness. It will flash to my specified setting every few seconds. I have no idea why it's broken and have attempted to reinstall it multiple times. Unfortunately, xflux is the only app that works for me.

**Comment by ntc2** [comment]
@bknie1 you probably already thought of this, but are you sure there are no other `xflux` or `redshift` processes running? If more than one of these programs are running at the same time, they could interfere with each other.

**Comment by bknie1** [comment]
@ntc2 So I didn't see any process conflicts. However, between investigating and switching GPU settings it seems to have fixed itself. A little OT but thanks for, at the very least, getting me to buckle down on it. I might be good? I'll go close my issue topic on Redshift now, haha. Thanks!

**Comment by mefzz** [comment]
@bknie1 Running working redshift here. Did you checked if any fluxgui process running in the background?

This solve my problem
> https://github.com/xflux-gui/xflux-gui/issues/27#issuecomment-247257554

**Comment by TeddyHartanto** [comment]
I'd love to see this implemented too! I like to use the tint no matter what time of the day it is.

**Comment by hippyk** [comment]
I have found the best solution: Install wine and use the windows version. Perceft run !

**Comment by ntc2** [comment]
Note from #52: this feature may be somewhat difficult to implement, because `xflux` itself -- the program that `fluxgui` uses to control the color temperature -- does not have this feature. So, it seems we would need to implement our own "is it day or night right now?" logic, duplicating that functionality in `xflux`, and then use the `xflux` feature that allows us to set the color temp independent of time of day. Of course, figuring out if it's day or night should be super easy using the latitude and longitude and some existing service; presumably `xflux` itself just calls out to some web service for this.

**Comment by johanbaaij** [comment]
@hippyk works great for me too except not in a dual monitor situation? did you happen to get that working? i'm using wine 2.13 (play on linux).

**Comment by hippyk** [comment]
@johanbaaij My situation is the same with you. But I don't think it's very bad. After all, only one monitor is used for most time.

**Comment by ntc2** [comment]
Note that Redshift already supports day time temps, so if we added Redshift backed per #113 it would solve this issue.

**Comment by Pipo05** [comment]
Hi everyone! i encountered the same issue trying to get the blue light removed during day time. Unlike in Windows, you just can't adjust the tint and have it stay for long. It only previews well and then returns to full brightness. 
So i used a little trick to get it working even daytime. That is, when it's daytime in the US, i know it's probably nighttime in places like some parts in Africa or Asia. So i'd put the coordinates(latitude/longitude) of a city in Africa or Asia  in the app's preferences. And it works fine for me. it would dim until it turns daytime in that part of the world, at which point it'll be nighttime in the US and simply replace the coordinates(latitude/longitude) with local US coordinates. (Also, when copying the coordinates from the google page that opens up, make sure to first part before comma for latitude and the other part for longitude; instead copying everything in one field of the preferences and then leaving the longitude field empty)Hope this helps someone until we find a permanent fix for Linux users.

**Comment by ntc2** [comment]
We now have Redshift support, so implementing this feature for the Redshift backend would be very simple if anyone wants to take a stab at it.
